lib/overcommit/hook/pre_commit/es_lint.rb

```diff
@@ -12,19 +12,30 @@ module Overcommit::Hook::PreCommit
   #     enabled: true
   #     command: ['npm', 'run', 'lint', '--', '-f', 'compact']
   #
+  # If ESLint returns some errors, you can bindly pass by
+  # setting `bindly_pass` option to true.
```

Comment:
In that case, I think it's better to remove it completely. If not having the configuration option turns out to be an issue for people, we can always add it at that point.